src/vs/platform/protocol/electron-main/protocolMainService.ts

```diff
@@ -104,14 +104,34 @@ export class ProtocolMainService extends Disposable implements IProtocolMainServ
 			}
 		}
 
+		// In OSS, evict resources from the memory cache in the renderer process
+		// Refs https://github.com/microsoft/vscode/issues/148541#issuecomment-2670891511
+		let CacheControlheaders: Record<string, string> | undefined;
+		if (!this.environmentService.isBuilt) {
```

Comment:
Should this be created outside this method and just picked up, since its somewhat static?

Reply:
makes sense